core/src/main/java/org/apache/sdap/mudrod/integration/LinkageIntegration.java

```diff
@@ -173,32 +171,32 @@ public JsonObject getIngeratedListInJson(String input) {
    * the similarities from different sources
    */
   public Map<String, List<LinkedTerm>> aggregateRelatedTermsFromAllmodel(String input) {
-    aggregateRelatedTerms(input, props.getProperty("userHistoryLinkageType"));
-    aggregateRelatedTerms(input, props.getProperty("clickStreamLinkageType"));
-    aggregateRelatedTerms(input, props.getProperty("metadataLinkageType"));
-    aggregateRelatedTermsSWEET(input, props.getProperty("ontologyLinkageType"));
+    aggregateRelatedTerms(input, MudrodConstants.USE_HISTORY_LINKAGE_TYPE);
```

Comment:
This code should be 
```
aggregateRelatedTerms(input, props.getProperty(MudrodConstants.USE_HISTORY_LINKAGE_TYPE));
```
same with three instance below

Reply:
This variable is not in the conf.properties file. It is just a constant.

Reply:
ACK